source/npm/util.js

```diff
@@ -105,7 +105,20 @@ exports.checkIgnoreStrategy = ({files}) => {
 
 	if (!files && !npmignoreExists) {
 		console.log(`
-		\n${chalk.bold.yellow('Warning:')} No ${chalk.bold.cyan('files')} field specified in ${chalk.bold.magenta('package.json')} nor is a ${chalk.bold.magenta('.npmignore')} file present. Having one of those will prevent you from accidentally publishing development-specific files along with your package's source code to npm. 
+		\n${chalk.bold.yellow('Warning:')} No ${chalk.bold.cyan('files')} field specified in ${chalk.bold.magenta('package.json')} nor is a ${chalk.bold.magenta('.npmignore')} file present. Having one of those will prevent you from accidentally publishing development-specific files along with your package's source code to npm.
 		`);
 	}
 };
+
+exports.registry = async (pkgManager, pkg) => {
```

Comment:
What is this about? It looks unrelated to a preview mode.

Reply:
Agreed to extract these changes as a separated PR. And better to rename this function as `registryUrlFromPkgManager` or `getRegistryUrl`.

Reply:
> What is this about? It looks unrelated to a preview mode.

https://github.com/sindresorhus/np/pull/399#issuecomment-531185309

> Agreed to extract these changes as a separated PR. And better to rename this function as `registryUrlFromPkgManager` or `getRegistryUrl`.

I agree

Reply:
PR created #485